lib/main_common.pm

```diff
@@ -2763,6 +2763,10 @@ sub load_hypervisor_tests {
         loadtest "virt_autotest/libvirt_isolated_virtual_network";
     }
 
+    if (check_var('VIRT_PART', 'irqbalance')) {
```

Comment:
This should be enabled for xen only, right?

Reply:
the qam-xen-irqbalance test suite is only listed in SLE-xxx Virt XEN Incidents job group. And the irqbalance test will return if it is not xen host. I did not add the check in module scheduling in case the test suite is added to kvm job group accidently.